02-front-end/11bloco-react-intro/01dia-intro/exercicio-11-1/src/Header.jsx

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+export default class Header extends Component {
+  render() {
+    return (
+      <div className='App-header'>
+        <h1>Conteúdos de Front-End</h1>
+      </div>
```

Comment:
De novo, algo mais semântico ficaria melhor. 
```suggestion
      <header className='App-header'>
        <h1>Conteúdos de Front-End</h1>
      </header>
```